ansible/roles/operator-pipeline/templates/openshift/tasks/publish-to-ocp-registry.yml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: publish-to-ocp-registry
+spec:
+  params:
+    - name: cert_project_id
+      description: ID of the bundle Certification Project (as in bundle ci.yaml)
+    - name: vendor_label
+      description: name of the vendor
+    - name: repository_name
+    - name: dist_method
+      description: distribution method- either marketplace or certified
+    - name: image
+      description: reference to the existing index image
+    - name: is_latest
+      description: If explicitly set to "true", resulting image will be tagged as "latest"
+    - name: bundle_version
+      description: Operator bundle version.
+  results:
+    - name: container_digest
+  steps:
+    - name: imagestream-manipulations
+      image: quay.io/openshift/origin-cli:4.7
```

Comment:
As we discuss this with Brian last week we should use only images that are owned by teams that work on certification workflow and are stored in a trusted registry under our control. Because the openshift folks can accidentally remove the tag and our pipeline start failing.